packages/shared-ux/prompt/no_data_views/impl/src/no_data_views.component.tsx

```diff
@@ -7,95 +7,231 @@
+            defaultMessage="Create data view"
+          />
+        </EuiButton>
+      )}
+      <EuiHorizontalRule />
+      <DocumentationLink href={dataViewsDocLink} data-test-subj="docLinkDataViews" />
+    </>
+  ) : undefined;
+
+  return (
+    <EuiCard
+      hasBorder={true}
+      data-test-subj="noDataViewsPromptCreateDataView"
+      display={emptyPromptColor}
+      {...{ icon, title, description, footer }}
+    />
+  );
+};
+
+const PromptTryEsql = ({
+  onTryESQL,
+  esqlDocLink,
+  emptyPromptColor,
+}: Pick<
+  NoDataViewsPromptComponentProps,
+  'onClickCreate' | 'onTryESQL' | 'esqlDocLink' | 'emptyPromptColor'
+>) => {
+  if (!onTryESQL) {
+    // we need to handle the case where the Try ES|QL click handler is not set because
+    // onTryESQL is set via a useEffect that has asynchronous dependencies
+    return null;
+  }
 
   // Load this illustration lazily
   const Illustration = withSuspense(
     React.lazy(() =>
-      import('./data_view_illustration').then(({ DataViewIllustration }) => {
-        return { default: DataViewIllustration };
+      import('./esql_illustration').then(({ EsqlIllustration }) => {
+        return { default: EsqlIllustration };
       })
     ),
-    <EuiPanel color="subdued" style={{ width: 226, height: 206 }} />
+    <EuiPanel color="subdued" style={{ width: 110, height: 100 }} />
   );
 
   const icon = <Illustration />;
-  const actions = (
-    <NoDataButtonLink {...{ onClickCreate, canCreateNewDataView, onTryESQL, esqlDocLink }} />
+
+  const title = (
+    <FormattedMessage
+      id="sharedUXPackages.noDataViewsPrompt.esqlPanel.title"
+      defaultMessage="Query your data with ES|QL"
+    />
+  );
+
+  const description = (
+    <FormattedMessage
+      id="sharedUXPackages.noDataViewsPrompt.esqlExplanation"
+      defaultMessage="ES|QL is a next-generation transformative piped query language and engine developed by Elastic. ES|QL simplifies workflows and advanced searches while accelerating query response for efficient data processing."
+    />
+  );
+
+  const footer = (
+    <>
+      <EuiButton onClick={onTryESQL} fill={true} data-test-subj="tryESQLLink">
+        <FormattedMessage
+          id="sharedUXPackages.noDataViewsPrompt.tryEsqlText"
+          defaultMessage="Try ES|QL"
+        />
+      </EuiButton>
+      <EuiHorizontalRule />
+      {esqlDocLink && <DocumentationLink href={esqlDocLink} data-test-subj="docLinkEsql" />}
+    </>
   );
 
   return (
-    <EuiEmptyPrompt
-      data-test-subj="noDataViewsPrompt"
-      layout="horizontal"
-      css={css`
-        max-width: ${MAX_WIDTH}px !important; // Necessary to override EuiEmptyPrompt to fit content
-        flex-grow: 0;
-      `}
-      color={emptyPromptColor}
-      {...{ actions, icon, title, body, footer }}
+    <EuiCard
+      hasBorder={true}
+      data-test-subj="noDataViewsPromptTryEsql"
+      display={emptyPromptColor}
+      betaBadgeProps={{
```

Comment:
@stratoula, this isn't technical preview now, or is it still?

Reply:
ES|QL is in GA from 8.11!

Reply:
Thanks for noticing this! I've pushed 4a70ee85dd7